bftengine/src/bftengine/ReplicaSpecificInfoManager.cpp

```diff
@@ -64,7 +64,9 @@ void RsiDataManager::init() {
     for (uint32_t offset = 0; offset < max_client_batch_size_; offset++) {
       std::vector<uint8_t> data = ps_->getReplicaSpecificInfo(clientId * max_client_batch_size_ + offset);
       if (data.empty()) continue;
-      clientRsiData.emplace_back(RsiItem::deserialize(data));
+      auto rsi_item = RsiItem::deserialize(data);
+      if (rsi_item.data().empty()) continue;
```

Comment:
Just curious how it could happen?

Reply:
In case we "remove" the RSI, which means simply setting it to empty string